File: src/main/java/logan/pickpocket/commands/ToggleCommand.java
package logan.pickpocket.commands;

import logan.api.command.BasicCommand;
import logan.api.command.SenderTarget;
import logan.pickpocket.config.MessageConfig;
import logan.pickpocket.config.Config;
import org.bukkit.entity.Player;

import java.util.Collections;

public class ToggleCommand extends BasicCommand<Player> {

    public ToggleCommand() {
        super("toggle", "pickpocket.admin.toggle", 0, 0,
                new String[0], SenderTarget.PLAYER, "pickpocket",
                Collections.emptyList(),
                "Usage:\n/pickpocket toggle");
    }

    @Override
    public boolean run(Player sender, String[] args, Object data) {
        boolean status = !Config.isPickpocketingEnabled();
        Config.setPickpocketingEnabled(status);
        sender.sendMessage(status ? MessageConfig.getPickpocketToggleOnMessage()
                : MessageConfig.getPickpocketToggleOffMessage());
        return true;
    }
}

File: src/logan/pickpocket/commands/ToggleCommand.kt
package logan.pickpocket.commands

import logan.api.command.BasicCommand
import logan.api.command.SenderTarget
import logan.pickpocket.config.MessageConfiguration
import logan.pickpocket.user.PickpocketUser
import org.bukkit.entity.Player

class ToggleCommand : BasicCommand<Player>(
    name = "toggle",
    parentCommand = "pickpocket",
    target = SenderTarget.PLAYER,
    permissionNode = "pickpocket.toggle",
    usage = """
        Usage:
        /pickpocket toggle (no args)
    """.trimIndent()
) {
    override fun run(sender: Player, args: Array<out String>, data: Any?): Boolean {

        val profile = PickpocketUser.get(sender)
        profile.isParticipating = !profile.isParticipating
        sender.sendMessage(
            if (profile.isParticipating)
                MessageConfiguration.pickpocketToggleOnMessage else
                MessageConfiguration.pickpocketToggleOffMessage
        )

        return true
    }
}